tensorflow_probability/python/mcmc/kernel.py

```diff
@@ -84,3 +84,22 @@ def bootstrap_results(self, init_state):  # pylint: disable=unused-argument
         `Tensor`s representing internal calculations made within this function.
     """
     return []
+
+  def copy(self, **override_parameter_kwargs):
+    """Non-destructively creates a deep copy of the kernel
+
+    Args:
+      override_parameter_kwargs: Python String/value `dictionary` of
+      initialization arguments to override with new values.
+
+    Returns:
+      new_kernel: A `TransitionKernel` object of same type as `self`,
+      initialized with override_parameter_kwargs
+    """
+    parameters = dict(self.parameters, **override_parameter_kwargs)
+    new_kernel = type(self)(**parameters)
+    # pylint: disable=protected-access
```

Comment:
you can put this pylint comment at the end of the line it applies to and avoid the need for the re-enable down below

Reply:
(this may not apply anymore after you fix the merge issue alexey notes below, but it's good to know anyway)

Reply:
Definitely good to know!